src/components/App.jsx

```diff
@@ -22,53 +22,55 @@ import PermissionsTab from 'components/Permissions/PermissionsTab'
 import IntentRedirect from 'services/IntentRedirect'
 import PermissionsApplication from 'containers/PermissionsApplication'
 import Permission from 'containers/Permission'
+import { Menu } from './pages/Menu'
```

Comment:
aliased path please. We should never use relative path because our override system doesn't work like that. 

Maybe we should create a lint rules for that?

Reply:
Yes, that error has happened to me one too many times. Always because of my IDE autocomplete, which sometimes figures out the aliased path, sometimes doesn't.

Reply:
I will implement https://github.com/MelvinVermeer/eslint-plugin-no-relative-import-paths into cozy-eslint. It will require that cozy-apps update the config via yarn once published, but that will ensure the issue never arise again

Reply:
Should be at least opt-out at first. Some applications have never been overriden so there is relative path everywhere.

Reply:
Noted